src/data/finset/lattice.lean

```diff
@@ -249,6 +249,14 @@ le_antisymm
 lemma sup_id_eq_Sup [complete_lattice α] (s : finset α) : s.sup id = Sup s :=
 by simp [Sup_eq_supr, sup_eq_supr]
 
+lemma sup_id_eq_sUnion {α} (s : finset (set α)) :
+  s.sup id = ⋃₀(↑s) :=
+sup_id_eq_Sup _
+
+@[simp] lemma sup_eq_bUnion {α β : Type*} {s : finset α} {f : α → set β} :
+  s.sup f = ⋃ x ∈ s, f x :=
+sup_eq_supr _ _
```

Comment:
To match the `inf` lemmas:
```suggestion
lemma sup_id_set_eq_sUnion (s : finset (set α)) : s.sup id = ⋃₀(↑s) :=
sup_id_eq_Sup _

@[simp] lemma sup_set_eq_bUnion (s : finset α) (f : α → set β) : s.sup f = ⋃ x ∈ s, f x :=
sup_eq_supr _ _
```

Reply:
OK, I only had spotted the change in the name, not in the arguments. (And it was too late for an automatic commit, I had made the modifications from bottom to top…)